ansible/roles/monitoring-statsd-fe-influxdb/tasks/main.yml

```diff
@@ -1,28 +1,37 @@
-- name: Stop statsd-fe-influxdb (if needed)
-  command: "docker stop statsd-fe-influxdb"
+- name: Create directories which need to exist if necessary
+  file: path={{ item }} state=directory owner=root
+  with_items:
+    - "{{ monitoring_tmp_dir }}"
   sudo: yes
-  ignore_errors: yes
   tags:
     - monitoring
     - statsd
     - statsd-fe-influxdb
 
-- name: Remove statsd-fe-influxdb (if there)
-  command: "docker rm statsd-fe-influxdb"
+- name: Download telegraf package
+  get_url: url="http://get.influxdb.org/telegraf/telegraf_{{ telegraf_version }}_amd64.deb" dest="{{ monitoring_tmp_dir }}"
```

Comment:
Can we make the url a variable that defaults to this address? Internally we should use registry.roger... like for influxdb (we'll need to keep the binary there ofcourse.)